config/jobs/image-pushing/k8s-staging-cluster-api.yaml

```diff
@@ -190,3 +190,32 @@ postsubmits:
               - --scratch-bucket=gs://k8s-staging-scl-image-builder-gcb
               - --env-passthrough=PULL_BASE_REF
               - .
+
+periodics:
+- name: cluster-api-push-images-nightly
+  cluster: k8s-infra-prow-build-trusted
+  decorate: true
+  interval: 24h # one day
```

Comment:
is there a way to trigger it at a certain time so it's actually nightly (for PST since that's when the least PRs merge)?

Reply:
https://github.com/kubernetes/test-infra/blob/master/prow/jobs.md#how-to-configure-new-jobs you can use `cron`

Reply:
Thanks @spiffxp 
Set to run midnight PT.